src/components/masthead/masthead.js

```diff
@@ -30,7 +29,6 @@ class Masthead extends React.Component {
       return;
     }
     logout().then(() => {
-      window.localStorage.clear();
```

Comment:
we want to clear local storage on logout for the user in mock mode since we are using the same user name for the reasons you said above.  For a real user we shouldn't clear the localStorage, each user has their own progress saved based on their username.  We might want to talk with UX and have a settings menu where we can do things like reset progress.   What do you think?

Reply:
ok sounds good